### ISSUE TITLE
Is there a way to set default language to translated one?

### ISSUE THREAD

**Issue by Gioware** [doublespeakgames/adarkroom#556]
Hello

is there a way to somehow set default language to something other than English? 

Thanks

**Comment by Continuities** [comment]
You can set it using URL parameters:
http://adarkroom.doublespeakgames.com/?lang=XX

**Comment by Gioware** [comment]
Yeah, I mean is it possible homepage to directly open with another language? to hide url parameters as well

**Comment by Continuities** [comment]
There is currently no support for this, but it would be relatively simple
to build some sort of persistence using localStorage.

**Comment by gitupofftheflooranddosomework** [comment]
Added to todo list

**Comment by angelofan** [comment]
> Yeah, I mean is it possible homepage to directly open with another language? to hide url parameters as well


https://github.com/doublespeakgames/adarkroom/blob/1fada4620b6c66bd07bf15a3f1eb8223df8bc1d7/index.html#L39

```
lang = localStorage.lang ?? 'zh_cn';
```
